docs/application/dotnet/guides/connectivity/bluetooth.md

```diff
@@ -1078,22 +1078,14 @@ To discover nearby LE devices, perform an LE scan operation:
             }
         }
 
-        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetServiceDataList();
+        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetGetServiceDataList(BluetoothLePacketType.BluetoothLeAdvertisingPacket);
```

Comment:
Would you please check this GetGetServiceDataList?

Reply:
Done. Thank you!